store/store_test.go

```diff
@@ -9,31 +10,23 @@ import (
 	"testing"
 	"time"
 
-	"github.com/globalsign/mgo"
-	"github.com/globalsign/mgo/bson"
+	"go.mongodb.org/mongo-driver/bson"
 
-	"github.com/tidepool-org/go-common/clients/mongo"
+	tpMongo "github.com/tidepool-org/go-common/clients/mongo"
 )
 
-var testingConfig = &mongo.Config{ConnectionString: "mongodb://localhost/data_test"}
+var testingConfig = &tpMongo.Config{ConnectionString: "mongodb://127.0.0.1/data_test", Database: "data_test"}
```

Comment:
why do we/they continue testing this old configuration schema? :/

Reply:
as far as I understand the new schema, having the _ConnectionString_ will ignore the other config items

Reply:
the go-common mongo config is mainly used to build the connection string based on the params (hosts,port,user,password...). And it can be initialized with the connection string. 

But in the main tide-whisperer package, it is initialized with the method FromEnv() that takes separate environments. So we now need them to run it. 

They just use this shortcut for unit testing the store package.